guide/english/wordpress/index.md

```diff
@@ -149,12 +149,12 @@ In Directory: -Leave Here Empty-
 
 ### Site Settings
 Site Name: My Blog
-Site Description: Awesome blog about cycling and sight seeing!
+Site Description: Awesome blog about cycling and sightseeing!
```

Comment:
replace `sightseeing` with `sight-seeing`, as this seems like the best way of representing such words